velox/core/PlanNode.h

```diff
@@ -545,6 +545,37 @@ class TableWriteNode : public PlanNode {
   const connector::CommitStrategy commitStrategy_;
 };
 
+class TableWriteMergerNode : public PlanNode {
```

Comment:
Would be nice to update documentation at https://facebookincubator.github.io/velox/develop/operators.html